### ISSUE TITLE
"should navigate to a dedicated build and to use the rebuild button" e2e test fails on webkit

### ISSUE THREAD

**Issue by p12tic** [buildbot/buildbot#7308]
After recent test flakiness fixes this test fails much more frequently than before. E.g. https://buildbot.buildbot.net/#/builders/123/builds/899.

```
[44/51] [webkit] › rebuilds.spec.ts:28:7 › rebuilds › should navigate to a dedicated build and to use the rebuild button
2) [webkit] › rebuilds.spec.ts:28:7 › rebuilds › should navigate to a dedicated build and to use the rebuild button
Error: went into build
expect(received).toBeTruthy()
Received: false
Call Log:
- Timeout 15000ms exceeded while waiting on the predicate
at pages/builder.ts:123
121 | }, {
122 | message: "went into build"
> 123 | }).toBeTruthy();
| ^
124 | }
125 |
126 | static async clickStopButton(page: Page) {
at Function.waitGoToBuild (/buildbot/buildbot-job/build/smokes-react/tests/pages/builder.ts:123:8)
at /buildbot/buildbot-job/build/smokes-react/tests/rebuilds.spec.ts:38:23
[45/51] [webkit] › settings.spec.ts:24:9 › manage settings › waterfall › change the "scalling factor" and check it
```

**Comment by p12tic** [comment]
Test disabled in https://github.com/buildbot/buildbot/pull/7352, should be re-enabled when issue is fixed.

**Comment by mokibit** [comment]
Tests were enabled in #7443.
This issue was fixed by #7434.
